package.json

```diff
@@ -55,6 +55,7 @@
     "@kitzen/assets": "*"
   },
   "devDependencies": {
+    "tronweb": "^5.3.0",
```

Comment:
This is for further testing purposes, as you see it's in dev section, thus it's not being installed on apps that use this lib